### ISSUE TITLE
Add accessibility performance

### ISSUE THREAD

**Issue by jendiamond** [projectblacklight/blacklight_range_limit#93]
There are some accessibility issues with this gem. I am working on some fixes now but they aren't perfected yet. :)

---

### The date range slider  form accessibility issues on our site

The form tag is not labelled but the slider is well identified and useable. I added an `aria-labelledby` to the [partial `_range_limit_pane`](https://github.com/projectblacklight/blacklight_range_limit/blob/v6.3.3/app/views/blacklight_range_limit/_range_limit_panel.html.erb) in our site but I think it needs to be a bit different.

![Screen Shot 2019-01-24 at 11.03.09 AM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5615719ec6cad148008cc8ae/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5700b77226456f76b7191a379ee0c2b52415f47bc30254d58eeb9cb74fce36f34985619e8836e0a20c73919a5e98216ee54ba208a1d83bfbb8c4dc5fa2137a5dad11b61435c52dcbd4f3fb29843c259f34fe94e50a2143365e573356e0a.png)

```
<div class="limit_content range_limit" aria-labelledby="range limit slider">
```

---

#### Possible fixes

+ Add a label https://www.w3.org/WAI/tutorials/forms/labels/

+ Use this gem to add a label to the form [WAIAble 1.0.0 released! Now you can make your Rails forms accessible.](https://accessiblerails.wordpress.com/)

**Comment by jrochkind** [comment]
Main branch and forthcoming 9.0 release has significantly revamped UI.  

It no longer features a slider at all, so this specific issue will no longer be relevant. 

It has various other tweaks intended to improve accessibility. If any problems still remain, please open another issue!
